tests/testthat/test-test-threads.R

```diff
@@ -113,11 +113,12 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
 
   test_that("check_run_times() returns NA for dry runs [BBR-TSTT-005]", {
+    skip_if_old_bbi('3.1.1')
     expect_message(
       check_run_times(mod1, .wait = F),
       "Could not access data for 1"
     )
-    run_times <- check_run_times(mod1, .wait = F) %>% suppressMessages()
+    run_times <- check_run_times(mod1, .wait = F) %>% suppressMessages() %>% suppressWarnings()
```

Comment:
Obviously minor, but this `suppressWarnings()` shouldn't be necessary due to the skip, right?

Reply:
hmm I think you're right but there was an instance where I had to do that. Maybe I hadn't added the skip yet or something. Definitely save to remove though

Reply:
Going to fix this since we'd want to see if any warnings occur (was there to silence a versioning warning, and I think I initially _didn't_ want to skip this test). Will have to re-request PR approval